specs/cli/pipelines-e2e.spec

```diff
@@ -0,0 +1,143 @@
+# Verify Pipelines
+
+Pre condition:
+  * Operator should be installed
+
+## Create and run Pipeline using cli (without resources)
+Tags: pipelines, e2e
+
+Steps:
+  * tkn create task "../../testdata/cli/shell-script-task.yaml"
+  * Verify taks creation status "successfull"
+  * tkn create task "../../testdata/cli/python-script-task.yaml"
+  * Verify task creation status "successfull"
+  * tkn pipeline create "../../testdata/cli/pipeline-script.yaml"
+  * tkn pipeline start "pipeline-test"
+  * tkn pipeline list
+  * Verify the status of pipeline "running"
+  * tkn taskrun list
+  * Verify the status of taskrun "running"
+  * Wait for the pipelinerun to complete
+  * Verify the pipelinerun status "successfull"
+
+
+## Create and run Pipeline using cli (with resources) (manual)
+Tags: pipelines, e2e
+
+Steps:
+  * tkn resource create "../../testdata/cli/git-resource.yaml"
+  * oc apply "../../testdata/cli/resource-condition.yaml"
+  * tkn create task "../../testdata/cli/resource-task.yaml"
+  * tkn pipeline create "../../testdata/cli/pipeline-condition.yaml"
+  * Start the pipeline manually by selecting the resource manually
+  * tkn pipeline list
+  * Verify the status of pipeline "running"
+  * Wait for pipelinerun to complete 
+  * Verify the pipelinerun status "successfull"
+
+
+## Create and run Pipeline using cli (with resources) (Automated)
+Tags: pipelines, e2e
+
+Steps:
+  * tkn resource create "../../testdata/cli/git-resource.yaml"
+  * oc apply "../../testdata/cli/resource-condition.yaml"
+  * tkn create task "../../testdata/cli/resource-task.yaml"
+  * tkn pipeline create "../../testdata/cli/pipeline-condition.yaml"
+  * tkn pipeline start "pipeline-to-list-files" "pipeline-git=ui-repo"
```

Comment:
table is mainly used when we want to test a single test case with multiple test data. Since it is having only one test data I did not add table for it.